src/providers/ldap/sdap_child_helpers.c

```diff
@@ -160,7 +159,7 @@ static errno_t create_tgt_req_send_buffer(TALLOC_CTX *mem_ctx,
         return ENOMEM;
     }
 
-    buf->size = 6 * sizeof(uint32_t);
+    buf->size = 7 * sizeof(uint32_t);
```

Comment:
Does this change need to be reflected in tests also?

Reply:
What test? (sure I could miss)

Reply:
Tests would fail already I suppose so disregard.